### ISSUE TITLE
[DPP] Unit tests failing for Data-Sovereignty policy check feature

### ISSUE THREAD

**Issue by matbmoser** [eclipse-tractusx/digital-product-pass#310]
## Description

I was integrating yesterday two features catenax-ng:release/v3.0.0-single-api and catenax-ng:release/v3.0.0-data-sovereignty
I found a bug related to the unit tests (PolicyUtilTest): 

```
IllegalState ApplicationContext failure threshold (1) exceeded
```

It is related to the unit test configuration that can not be loaded during compilation.

## Possible solutions
- Problem with the annotations in the Unit Test
- Problem with the configuration resources

**Comment by matbmoser** [comment]
So I updated and merged today a fix:

- Removed functionality from EDC util test to policy util, since there we are really testing the policies
- Added the test application yaml in the pom resources.
- Removed the context annotation location from the annotation.

**Comment by matbmoser** [comment]
Fix added to #308  and #306 PRs 🚀
